index.html

```diff
@@ -8,66 +8,73 @@
     <link href='css/skeleton.css' type='text/css' rel='stylesheet'>
     <link href='css/base.css' type='text/css' rel='stylesheet'>
     <link href='css/layout.css' type='text/css' rel='stylesheet'>
-    <link href='css/icon.css' type='' rel='stylesheet'>
     <link rel="stylesheet" href="https://cdn.linearicons.com/free/1.0.0/icon-font.min.css">
     <link href='https://fonts.googleapis.com/css?family=Lobster|Architects+Daughter' rel='stylesheet' type='text/css'>
 
 
   </head>
   <body>
-    <div id='wrapper'>
-      <header class='navbar'>
-        <h1 class='myname'>Lacey Vu</h1>
-          <nav class='navbar' >
-            <ul>
-              <li id='mainpage'>Home</li>
-              <li id="about-me">About</li>
-              <li class='projectlink' id="projecthistory" >Projects</a></li>
-              <li class='employed' id='employedhistory'>Employment</li>
-          </ul>
-        </nav>
-    </header>
-    <section class='personalinfo' class='four columns row'>
-      <img src='imgs/lacey.jpg' class='photoMe'>
-        <ul class='social media'>
-          <a href='https://www.facebook.com/lacelin2010'><img src='imgs/facebook.png'></a>
-          <a href='https://twitter.com/LaceyVu'><img src='imgs/twitter.png'></a>
-          <a href='https://www.linkedin.com/profile/view?id=AAMAABFJIagBqvNwtZ8994EGUah_0nwhuXFxaRA&trk=hp-identity-photo'><img src='imgs/linkedin.png'></a>
-          <a href='https://github.com/laceylin2010'><img src='imgs/github.jpg'></a>
-        </ul>
-    <section id='aboutme' class='bio' >
-      <h1>About Me</h1>
-        <p>My name is Lacey and I have an Associates in Business from Highline Community College. Later down the years, I found out that I did not know what to exactly do with my degree and ended up working a bunch of jobs that did not relate as much instead. I heard of code fellows from a friend and decided to give it a try. After taking 201 I have realized how much I enjoy it and how awesome it would be to turn into a career. So here goes!</p>
+      <section class='five columns'>
+        <header class='navbar'>
+          <h1 class='myname'>Lacey Vu</h1>
+          <img src='imgs/lacey.jpg' class='photome'>
+            <nav class='navbar' >
+              <ul class='social media'>
+                <a href='https://www.facebook.com/lacelin2010'><img src='imgs/facebook.png'></a>
+                <a href='https://twitter.com/LaceyVu'><img src='imgs/twitter.png'></a>
+                <a href='https://www.linkedin.com/profile/view?id=AAMAABFJIagBqvNwtZ8994EGUah_0nwhuXFxaRA&trk=hp-identity-photo'><img src='imgs/linkedin.png'></a>
+                <a href='https://github.com/laceylin2010'><img src='imgs/github.jpg'></a>
+              </ul>
+              <ul class='menubar'>
+                <img src='imgs/hamburger.png' class='hamburger'>
+                <li id='mainpage'>Home</li>
```

Comment:
@laceylin2010 try to be a bit more semantic and us similar naming conventions on your classes/id's - for example, you have:

id="mainpage"
id="about-me"

This is more clear and semantic:

id="main-page" or id="main_page"
id="about-me" or id="about_me"